exchanges/gateio/gateio_test.go

```diff
@@ -74,838 +75,3066 @@ func TestStart(t *testing.T) {
+		t.Errorf("%s GetDeliveryOrders() error %v", g.Name, err)
+	}
+}
+
+func TestCancelAllDeliveryOrders(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.CancelMultipleDeliveryOrders(context.Background(), deliveryFuturesTradablePair, "ask", settleUSDT); err != nil && !strings.Contains(err.Error(), "USER_NOT_FOUND") {
+		t.Errorf("%s CancelAllDeliveryOrders() error %v", g.Name, err)
+	}
+}
+
+func TestGetSingleDeliveryOrder(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.GetSingleDeliveryOrder(context.Background(), settleUSDT, "123456"); err != nil && !strings.Contains(err.Error(), "ORDER_NOT_FOUND") {
+		t.Errorf("%s GetSingleDeliveryOrder() error %v", g.Name, err)
+	}
+}
+
+func TestCancelSingleDeliveryOrder(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.CancelSingleDeliveryOrder(context.Background(), settleUSDT, "123456"); err != nil && !strings.Contains(err.Error(), "ORDER_NOT_FOUND") {
+		t.Errorf("%s CancelSingleDeliveryOrder() error %v", g.Name, err)
+	}
+}
+
+func TestGetDeliveryPersonalTradingHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.GetDeliveryPersonalTradingHistory(context.Background(), settleUSDT, "", deliveryFuturesTradablePair, 0, 0, 1, ""); err != nil && !strings.Contains(err.Error(), "CONTRACT_NOT_FOUND") {
+		t.Errorf("%s GetDeliveryPersonalTradingHistory() error %v", g.Name, err)
+	}
+}
+
+func TestGetDeliveryPositionCloseHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.GetDeliveryPositionCloseHistory(context.Background(), settleUSDT, deliveryFuturesTradablePair, 0, 0, time.Time{}, time.Time{}); err != nil && !strings.Contains(err.Error(), "CONTRACT_NOT_FOUND") {
+		t.Errorf("%s GetDeliveryPositionCloseHistory() error %v", g.Name, err)
+	}
+}
+
+func TestGetDeliveryLiquidationHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.GetDeliveryLiquidationHistory(context.Background(), settleUSDT, deliveryFuturesTradablePair, 0, time.Now()); err != nil {
+		t.Errorf("%s GetDeliveryLiquidationHistory() error %v", g.Name, err)
+	}
+}
+
+func TestGetDeliverySettlementHistory(t *testing.T) {
+	t.Parallel()
+	if !areTestAPIKeysSet() {
+		t.Skip(credInformationNotProvided)
+	}
+	if _, err := g.GetDeliverySettlementHistory(context.Background(), settleUSDT, deliveryFuturesTradablePair, 0, time.Now()); err != nil {
+		t.Errorf("%s GetDeliverySettlementHistory() error %v", g.Name, err)
+	}
+}
+
+func TestGetDeliveryPriceTriggeredOrder(t *testing.T) {
```

Comment:
GateIO GetDeliveryPriceTriggeredOrder() error GateIO unsuccessful HTTP status code: 400 raw response: {"label":"AUTO_INVALID_PARAM_TRIGGER_PRICE","message":"invalid argument: trigger.price price is not an integer multiple of a price unit"}


Though Im not exactly sure what the issue is here